installation/sql/mysql/joomla.sql

```diff
@@ -1994,6 +1994,7 @@ CREATE TABLE IF NOT EXISTS `#__users` (
   `otpKey` varchar(1000) NOT NULL DEFAULT '' COMMENT 'Two factor authentication encrypted keys',
   `otep` varchar(1000) NOT NULL DEFAULT '' COMMENT 'One time emergency passwords',
   `requireReset` tinyint(4) NOT NULL DEFAULT 0 COMMENT 'Require user to reset password on next login',
+  `access` int(10) unsigned NOT NULL DEFAULT 1,
```

Comment:
We should disable access to the users initially. Like that we do not run into security issues. I would default it to 0.